__tests__/store/ui.spec.js

```diff
@@ -96,23 +110,9 @@ describe('store/ui', () => {
 			});
 		});
 
-		it('return namespace page with namespace name', async () => {
-			// Arrange:
-			const namespaceName = 'symbol.xym';
-
-			const getNamespaceInfoStub = stub(NamespaceService, 'getNamespaceInfo');
-			getNamespaceInfoStub.returns(Promise.resolve({}));
-
-			// Act:
-			await ui.actions.search({ dispatch, rootGetters }, namespaceName);
+		it('return namespace page with namespace name', async () => await assertSearchNamespace('symbol.xym'));
 
-			// Assert:
-			expect(dispatch).toHaveBeenCalledTimes(1);
-			expect(dispatch).toHaveBeenNthCalledWith(1, 'openPage', {
-				pageName: 'namespace',
-				param: namespaceName
-			});
-		});
+		it('return namespace page with namespace name contain dash', async () => await assertSearchNamespace('symbol-'));
```

Comment:
should you have test with keyword that includes whitespace ?

Reply:
I missed that part, let me add it.

Reply:
I had a minor refactor on the code, hope it make sense